packages/schemas/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@message-queue-toolkit/schemas",
-  "version": "3.0.0",
+  "version": "4.0.0",
```

Comment:
why bump semver major?

Reply:
The `handler` type ha a new required variable that must be filled, it implies a breaking change

Reply:
but why does that affect schemas library? nothing changed for it